ldms/src/ldmsd/ldmsd_prdcr.c

```diff
@@ -325,7 +325,7 @@ static void __update_set_info(ldmsd_prdcr_set_t set, ldms_dir_set_t dset)
 
 		/* Sanity check the hints */
 		if (offset_us >= intrvl_us) {
-			ovis_log(prdcr_log, OVIS_LERROR, "set %s: Invalid hint '%s', ignoring hint\n",
+			ovis_log(prdcr_log, OVIS_LDEBUG, "set %s: hint '%s', ignoring hint\n",
```

Comment:
@nichamon, given that we removed the setting of the hint in ``ldmsd_set_register``, should we make this INFO instead of DEBUG?

Reply:
I think so. This will provide a clue when an updater with auto-interval enabled doesn't update a set. I'll update the patch.